htsget.md

```diff
@@ -175,7 +175,7 @@ _optional 32-bit unsigned integer_
 </td><td>
 The start position of the range on the reference, 0-based, inclusive. 
 
-The server SHOULD respond with an `InvalidInput` error if `start` is specified and a reference is not specified (see `referenceName`).
+The server SHOULD respond with an `InvalidInput` error if `start` is specified and a reference is unspecified or * (see `referenceName`).
```

Comment:
same as before

Reply:
Quotation marks around `*` here, and below too.